src/corehost/cli/comhost/CMakeLists.txt

```diff
@@ -40,4 +40,5 @@ if (WIN32 AND (CLI_CMAKE_PLATFORM_ARCH_ARM OR CLI_CMAKE_PLATFORM_ARCH_ARM64))
     target_link_libraries(comhost Advapi32.lib Ole32.lib OleAut32.lib)
 endif()
 
-install_library_and_symbols (comhost)
\ No newline at end of file
+install(TARGETS comhost DESTINATION corehost)
+install_symbols(comhost corehost)
```

Comment:
Can we move this to `lib.cmake` similarly to how it's in `exe.cmake` ?

Reply:
We could, but because we will have to add an additional install command in `ijwhost` anyway to install the `ijwhost.lib` file to the output directory, I prefer to keep the installs in the same file so they won't be split (and the main ones hidden in lib.cmake).

Reply:
OK - makes sense.